docs/web/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: ONNX Runtime Web
+description: ONNX Runtime for web-based deploymentss
+has_children: true
+has_toc: false 
+nav_order: 9
+
+---
+
+# ORT Web
```

Comment:
Same comment as for mobile - I would move the web overview and tutorials into here